cmd/telegraf/telegraf.go

```diff
@@ -315,6 +315,18 @@ func (t *Telegraf) runAgent(ctx context.Context, c *config.Config, reloadConfig
 		log.Printf("W! Deprecated secretstores: %d and %d options", count[0], count[1])
 	}
 
+	// Compute the amount of locked memory needed for the secrets
+	log.Printf("I! Managed secrets: %d", c.NumberSecrets)
```

Comment:
I do wonder what users, who are not using secret store are going to think when they see this show up :\ while it aids in debugging it might also lead to confusion

Reply:
So you want me to remove it?

Reply:
I think it would be good to remove for now and lets see where these fixes land us? Are you good with that?